web/src/test/manual/build.sh

```diff
@@ -19,22 +19,24 @@ builder_parse "$@"
 DEST="web/build/test-resources"
 
 builder_describe_outputs \
-  build       "/$DEST/sentry-manager.js"
+  build       "${KEYMAN_ROOT}/${DEST}/sentry-manager.js"
 
 #### Build action definitions ####
 
 function do_copy() {
-  mkdir -p "$KEYMAN_ROOT/$DEST/keyboards"
+  mkdir -p "${KEYMAN_ROOT}/${DEST}/keyboards"
 
   # The next two lines are needed for the sentry-integration manual test page.
-  cp "$KEYMAN_ROOT/common/web/sentry-manager/build/lib/index.js"      "$KEYMAN_ROOT/$DEST/sentry-manager.js"
-  cp "$KEYMAN_ROOT/common/web/sentry-manager/build/lib/index.js.map"  "$KEYMAN_ROOT/$DEST/sentry-manager.js.map"
+  cp "${KEYMAN_ROOT}/common/web/sentry-manager/build/lib/index.js"      "${KEYMAN_ROOT}/${DEST}/sentry-manager.js"
+  cp "${KEYMAN_ROOT}/common/web/sentry-manager/build/lib/index.js.map"  "${KEYMAN_ROOT}/${DEST}/sentry-manager.js.map"
+
+  cp "${KEYMAN_ROOT}/common/web/types/tests/fixtures/kmx/khmer_angkor.kmx" "${KEYMAN_ROOT}/${DEST}/"
```

Comment:
Why is this .kmx copied to `${DEST}` whereas the .js keyboards below are copied to `${DEST}/keyboards`?

Reply:
Good question. Probably oversight. Done.